### DISCUSSION TITLE
Default value: Should it appear in the template?

### DISCUSSION THREAD

**Discussion by carstenjaksch** [statamic/cms#11236] [Troubleshooting] [answered]
Hi,

I have a field that conditionally hides if another field's value does not match a certain condition. The hidden field has a default value.

Now if the field is hidden and the page saved, Statamic puts a `null` value in the frontmatter. However, the default value is now in the template.

Is this a bug or a feature? My understanding is that a default value should only populate an empty field in the CMS, if it is loaded the first time. Normally, such default values don't get to the template if the field is empty (in this case `null`). In that case, it would simply evaluate as falsy.

The behavior I encountered confused me. 😆

**Comment by edalzell** [comment, marked as answer]
Default values are available in the template now, regardless of if they're in the data. That change was made a while back.

**Reply by carstenjaksch** [reply]
Thanks, good to know!
